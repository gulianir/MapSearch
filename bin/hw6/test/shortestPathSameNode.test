# Load graph 
LoadGraph harryPotter harryPotter.tsv

# find path 
FindPath harryPotter Harry Harry 