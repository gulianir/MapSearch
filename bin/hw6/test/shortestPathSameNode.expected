# Load graph 
loaded graph harryPotter

# find path 
path from Harry to Harry: